.github/workflows/ci.yaml

```diff
@@ -0,0 +1,83 @@
+name: Docker image build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      MAILROOM_REPO: nyaruka/mailroom
+      GREP_TIMEOUT: 60
+    strategy:
+      matrix:
+        MAILROOM_VERSION: ["6.0.3", "6.2.3", "6.4.3", "7.0.1"]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker/setup-buildx-action@v1
+      - name: Run Redis
+        run: docker run --name redis -d -p 6379:6379 redis:3.2.4
+      - name: Install PostgreSQL
+        uses: nyaruka/postgis-action@v2
+        with:
+          postgresql version: "13"
+          postgis version: "3.1"
+          postgresql db: temba
+          postgresql user: temba
+          postgresql password: temba
+      - name: Build image
+        uses: docker/build-push-action@v2
+        with:
+          push: false
+          load: true
+          context: .
+          tags: mailroom
+          build-args: |
+            MAILROOM_REPO=${{env.MAILROOM_REPO}}
+            MAILROOM_VERSION=${{matrix.MAILROOM_VERSION}}
+      - name: Test image
+        run: |
+          docker run --name mailroom --env-file docker.envfile --link postgis --link redis --publish 8090:8090 --detach mailroom
+          timeout ${{env.GREP_TIMEOUT}} grep -m 1 'mailroom started' <(docker logs --follow mailroom 2>&1)
+          curl http://localhost:8090/ | grep -m 1 'mailroom'
+          curl http://localhost:8090/ | grep -m 1 ${{matrix.MAILROOM_VERSION}}
+      - name: debug outputs
+        if: always()
```

Comment:
Do we need this?

Reply:
This means that if there's a failure in the previous step (eg. one of the tests fails), then we can still view the logs and other details, which is useful